pkg/meta/random_test.go

```diff
@@ -932,7 +934,7 @@ func (m *fsMachine) listxattr(inode Ino) ([]byte, syscall.Errno) {
 
 func (m *fsMachine) Mkdir(t *rapid.T) {
 	parent := m.pickNode(t)
-	name := rapid.StringN(1, 200, 255).Draw(t, "name")
+	name := rapid.StringN(3, 200, 255).Draw(t, "name")
```

Comment:
why we need this change?

Reply:
already skip . and .. in the next judge, forget to convert it